src/tachimawari/control/controller/module/cm740.cpp

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "tachimawari/control/controller/module/cm740.hpp"
+
+#include "tachimawari/control/controller/module/cm740_address.hpp"
+#include "tachimawari/control/packet/protocol_1/instruction/bulk_read_packet.hpp"
+#include "tachimawari/control/packet/protocol_1/instruction/instruction.hpp"
+#include "tachimawari/control/packet/protocol_1/instruction/sync_write_packet.hpp"
+#include "tachimawari/control/packet/protocol_1/instruction/write_packet.hpp"
+#include "tachimawari/control/packet/protocol_1/model/packet_id.hpp"
+#include "tachimawari/control/packet/protocol_1/status/bulk_read_data.hpp"
+#include "tachimawari/control/packet/protocol_1/status/status_packet.hpp"
+#include "tachimawari/control/packet/protocol_1/utils/word.hpp"
+#include "tachimawari/joint/model/joint.hpp"
+#include "tachimawari/joint/protocol_1/mx28_address.hpp"
+
+#include "errno.h"  // NOLINT
+#include "fcntl.h"  // NOLINT
+#include "stdio.h"  // NOLINT
+#include "string.h"  // NOLINT
+#include "termios.h"  // NOLINT
+#include "unistd.h"  // NOLINT
+#include "linux/serial.h"
+#include "sys/ioctl.h"
+#include "sys/time.h"
```

Comment:
Why it's being reported error by the linter? As it's system header, have you tried writing it as `<errno.h>` to prevent linter error?